envi/tests/test_arch_arm.py

```diff
@@ -1732,66 +1957,96 @@ def test_envi_arm_assorted_instrs(self):
         goodcount = 0
         goodemu = 0
         bademu = 0
-        for archz, bytez, va, reprOp, iflags, emutests in instrs:
+        for tidx, testinfo in enumerate(instrs):
+            archz, bytez, va, reprOp, iflags, emutests = testinfo
             ranAlready = False  # support for run once only
-            #itterate through architectures 
+            #iterate through architectures 
             for key in ARCH_REVS:
                 test_arch = ARCH_REVS[key]
                 if ((not ranAlready) or (not self.armTestOnce)) and ((archz & test_arch & self.armTestVersion) != 0):
-                    ranAlready = True
-                    op = vw.arch.archParseOpcode(binascii.unhexlify(bytez), 0, va)
-                    redoprepr = repr(op).replace(' ','').lower()
-                    redgoodop = reprOp.replace(' ','').lower()
-                    if redoprepr != redgoodop:
-                        num, = struct.unpack("<I", binascii.unhexlify(bytez))
-                        bs = bin(num)[2:].zfill(32)
-                        badcount += 1
-                        raise Exception("%d FAILED to decode instr:  %.8x %s - should be: %s  - is: %s" % \
-                                (goodcount, va, bytez, reprOp, repr(op) ) )
-                        # self.assertEqual((goodcount, bytez, redoprepr), (goodcount, bytez, redgoodop))
+                    try:
+                        if verbose:
+                            print("%s: %r"% (tidx, testinfo))
```

Comment:
print -> log?

Reply:
done

Reply:
i don't think so...  it's a logger call in my current version? it's committed and pushed.  perhaps you're only looking at some of the commits?  this says outdated for me, and when i clicked on the file, it took me to only "20 commits" instead of "all commits"